core/tests/protractor/embedding.js

```diff
@@ -22,11 +22,66 @@ var users = require('../protractor_utils/users.js');
 var AdminPage = require('../protractor_utils/AdminPage.js');
 var editor = require('../protractor_utils/editor.js');
 var ExplorationPlayerPage =
-  require('../protractor_utils/ExplorationPlayerPage.js');
+  require('../protractor_utils/ExplorationPlayerPage.js')
+var workflow = require('../protractor_utils/workflow.js');
 
 describe('Embedding', function() {
   var adminPage = null;
   var explorationPlayerPage = null;
+  var explorationId = null;
+  var createCountingExploration = function (){
+    // Intro
+    editor.setStateName('Intro');
+    editor.setContent(forms.toRichText(
+      'Given three balls of different colors. How many ways are there ' +
+      'to arrange them in a straight line?'
+      )
```

Comment:
Move this to the start of the next line; in general, match the indentation between opening and closing parens.

Reply:
done.